### PR TITLE
routing settings

### PR DESCRIPTION
**Pull request by 47codemonkey**
Пока только пути прописал. Компоненты сейчас буду создавать в этой же ветке

### CODE REVIEW

**Comment by 47codemonkey** [comment]
Порезолвил твои конфликты

File: src/features/AboutProject/index.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    return (
+        <ul>
+        <li>
+          <Link to={PLANET_OBSERVER_PATH}>Planet Observer</Link>
+        </li>
+      </ul>  
+    )
+}
+
+export default AboutProject
```
Comment (dmytro-honch) [inline src/features/AboutProject/index.tsx]: You have to apply prettier changes onSave action in VS code. So, here should be semicolon.

File: src/features/AboutProject/index.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { Link } from 'react-router-dom'
+import { PLANET_OBSERVER_PATH } from '../Router/pathes.ts';
```
Comment (dmytro-honch) [inline src/features/AboutProject/index.tsx]: Please, use absolute path in imports 'src/features/Router...'

File: src/features/AboutProject/index.tsx (hunk 1 above)
Comment (dmytro-honch) [reply src/features/AboutProject/index.tsx]: This changes for whole PR.

File: src/features/AboutProject/index.tsx (hunk 2 above)
Comment (dmytro-honch) [reply src/features/AboutProject/index.tsx]: This changes for whole PR.

File: src/features/Router/index.tsx [hunk 3]
```diff
@@ -3,18 +3,26 @@ import { BrowserRouter, Route, Routes } from 'react-router-dom';
 import ObserverPage from 'src/features/ObserverPage';
 import MenuPage from 'src/features/Menu';
 import Page404 from 'src/features/404';
+import ContactUs from 'src/features/ContactUs';
+import AboutProject from 'src/features/AboutProject';
+import SolarSystemObjects from 'src/features/SolarSystemObjects';
 
-import { MENU_PATH, PLANET_OBSERVER_PATH } from './pathes';
+import { MENU_PATH, PLANET_OBSERVER_PATH, FALLBACK_404_ROUTE, CONTACT_US_PATH, ABOUT_US_PATH, SOLAR_SYSTEM_PATH } from './pathes.ts';
```
Comment (dmytro-honch) [inline src/features/Router/index.tsx]: don't use file extension in paths for ts/tsx/js/jsx files

File: src/features/Menu/index.tsx [hunk 4]
```diff
@@ -1,15 +1,38 @@
...
+        <Link to={SOLAR_SYSTEM_PATH}>Solar System objects</Link>
+      </li>
+      <li>
+        <Link to={ABOUT_US_PATH}>About us</Link>
+      </li>
+      <li>
+        <a href={PROPOSE_CHANGES_PATH}>Propose changes</a>
+      </li>
+      <li>
+        <a href={SAY_THANKS_PATH}>Say thanks</a>
```
Comment (dmytro-honch) [inline src/features/Menu/index.tsx]: well done :)

**Review by dmytro-honch — CHANGES_REQUESTED** [review]

File: src/features/ContactUs/index.tsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import { Link } from 'react-router-dom'
+import { PLANET_OBSERVER_PATH } from '../Router/pathes.ts';
```
Comment (dmytro-honch) [inline src/features/ContactUs/index.tsx]: Please, rename filename "pathes" with "path". My typo :)

**Comment by 47codemonkey** [comment]
Hey there. Kindly check my last commit "some minor changes' Thx in advance

File: src/features/SolarSystemObjects/index.tsx [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import { Link } from 'react-router-dom'
+import { PLANET_OBSERVER_PATH } from '../Router/pathes.ts';
```
Comment (dmytro-honch) [inline src/features/SolarSystemObjects/index.tsx]: use absolute path

File: src/features/Menu/index.tsx [hunk 7]
```diff
@@ -1,15 +1,38 @@
...
-      <Link to={'/unlivable_url'}>404</Link>
-    </li>
-  </ul>
-);
+import { PLANET_OBSERVER_PATH } from 'src/features/Router/pathes.ts';
+import { CONTACT_US_PATH } from 'src/features/Router/pathes.ts';
+import { ABOUT_US_PATH } from 'src/features/Router/pathes.ts';
+import { SOLAR_SYSTEM_PATH } from 'src/features/Router/pathes.ts';
+import { PROPOSE_CHANGES_PATH } from 'src/features/Router/pathes.ts';
+import { SAY_THANKS_PATH } from 'src/features/Router/pathes.ts';
```
Comment (dmytro-honch) [inline src/features/Menu/index.tsx]: delete file extensions

File: src/features/ContactUs/index.tsx [hunk 8]
```diff
@@ -0,0 +1,15 @@
+import { Link } from 'react-router-dom';
+import { PLANET_OBSERVER_PATH } from 'src/features/Router/pathes.ts';
```
Comment (dmytro-honch) [inline src/features/ContactUs/index.tsx]: delete file extension

File: src/features/AboutProject/index.tsx [hunk 9]
```diff
@@ -0,0 +1,14 @@
+import { Link } from 'react-router-dom';
+import { PLANET_OBSERVER_PATH } from 'src/features/Router/pathes.ts';
```
Comment (dmytro-honch) [inline src/features/AboutProject/index.tsx]: delete file extension

**Review by dmytro-honch — CHANGES_REQUESTED** [review]

**Review by dmytro-honch — APPROVED** [review]

**Pull request merged**